sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -378,6 +378,8 @@ class AfterCount(TriggerFn):
   COUNT_TAG = _CombiningValueStateTag('count', combiners.CountCombineFn())
 
   def __init__(self, count):
+    if not isinstance(count, int) or count < 1:
```

Comment:
<!--thread_id:cc_158342029_t; commit:2df1e910f7e7508a6253afeb297a7487f5d5c392; resolved:1-->
<!--section:context-quote-->
> **mariapython** wrote:
> It makes sense. I have just incorporated the right change so that long types are allowed while enforcing non-fractional numbers.

<!--section:body-->
This will still fail for float types like `3.0` but I think that is fine, we do not need to make more changes here.